### ISSUE TITLE
Link to template-use examples when available

### ISSUE THREAD

**Issue by davidlmobley** [livecomsjournal/article_templates#6]
As soon as we have examples using the templates (beyond just the simple example the template itself provides) we should link to them in the md files here.

**Comment by davidlmobley** [comment]
Should be able to link to "Why LiveCoMS" when up.

**Comment by davidlmobley** [comment]
@dmzuckerman - is your article at the point where we can link to it as an example of use of the template? Or @dwsideriusNIST ?

**Comment by dmzuckerman** [comment]
Probably ... but ironically I'm not sure if our checklist itself (that section) uses the template!  What do you think @dwsideriusNIST ?  FYI we still have editing a-plenty to do.  @davidlmobley Dan S did put a [compiled pdf](https://github.com/dmzuckerman/Sampling-Uncertainty/blob/master/main.pdf) there so you can check it out yourself.

**Comment by dwsideriusNIST** [comment]
Yeah, our checklist is not up to spec yet, so it needs to be upgraded. I've not personally worked on that section - Paul and I are still hashing out the definitions section.

**Comment by dmzuckerman** [comment]
I did update it in the last week.  Formatting is not beautiful for list sub-levels (same as main level) ... I made up for it by bolding main level
